devito/interfaces.py

```diff
@@ -4,11 +4,12 @@
 from tools import aligned
 import os
 from signal import signal, SIGABRT, SIGINT, SIGSEGV, SIGTERM
+from tempfile import gettempdir
 import sys
 import atexit
 
 
-__all__ = ['DenseData', 'TimeData', 'PointData']
+__all__ = ['DenseData', 'TimeData', 'PointData', 'MemmapManager', 'SymbolicData']
```

Comment:
No, please don't expose the abstract base classes to the user API. They are not intended for users, since they are non-trivial to handle and get right and should thus only be used by developers.